tests/test_projectbuilder.py

```diff
@@ -132,8 +132,8 @@ def test_check_dependencies(mocker, pyproject_mock):
     builder.hook.get_requires_for_build_wheel.side_effect = copy.copy(side_effects)
 
     # requires = []
```

Comment:
This comment appears to be inaccurate — it seems that `pyproject_mock` has `requires=['flit-core']`.